### ISSUE TITLE
INFO: pip is looking at multiple versions of nvidia-cufft-cu12 to determine which version is compatible with other requirements. This could take a while.

### ISSUE THREAD

**Issue by aschmitz13** [jhj0517/Whisper-WebUI#107]
i am unable to install it on several computers. I am using python 3.10. Pip runs in loops with errors similar to:

"INFO: pip is looking at multiple versions of nvidia-cufft-cu12 to determine which version is compatible with other requirements. This could take a while."

the package changes.

**Comment by jhj0517** [comment]
Hi. I don't know why, but it most likely seems to be a pip problem..
Could you try 
```
pip install --upgrade pip
```
and sharing full cmd messages during the installation would be helpful.

**Comment by tingtongdpnw** [comment]
its not a bug. @aschmitz13 you have to do following command: "alias python=python3"
because if you look in the install.sh file it says pip and python, thats why you get that multiple version info. You could also change that in the file to python3 and then it should work, do that everything clean again. @jhj0517 now to you mister minister from uganda. Please i need a solution for the part on how to set an custom domain instead of localhost, so lets move back to the other post. can you just share me an example where you do it for example for the domain "pingpong.com" or other example name. I just couldnt figure it out and show me exactly where you put that config change in which file (probably app.py file).

**Comment by xxetest** [comment]
there is dependency issue, it keeps downloading torch:
```
└─$ ./Install.sh
Looking in indexes: https://pypi.org/simple, https://download.pytorch.org/whl/cu118
Collecting git+https://github.com/jhj0517/jhj0517-whisper.git (from -r requirements.txt (line 3))
  Cloning https://github.com/jhj0517/jhj0517-whisper.git to /tmp/pip-req-build-t9b6548z
  Running command git clone --filter=blob:none --quiet https://github.com/jhj0517/jhj0517-whisper.git /tmp/pip-req-build-t9b6548z
  Resolved https://github.com/jhj0517/jhj0517-whisper.git to commit 9e4ad0cb345135de716275ce5f7cc36d94ecd88b
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting torch (from -r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/cu118/torch-2.2.1%2Bcu118-cp311-cp311-linux_x86_64.whl (819.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 819.2/819.2 MB 1.9 MB/s eta 0:00:00
Collecting faster-whisper==1.0.1 (from -r requirements.txt (line 4))
  Downloading faster_whisper-1.0.1-py3-none-any.whl.metadata (14 kB)
Collecting transformers (from -r requirements.txt (line 5))
  Downloading transformers-4.38.2-py3-none-any.whl.metadata (130 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 130.7/130.7 kB 828.6 kB/s eta 0:00:00
Collecting gradio==4.14.0 (from -r requirements.txt (line 6))
  Downloading gradio-4.14.0-py3-none-any.whl.metadata (15 kB)
Collecting pytube (from -r requirements.txt (line 7))
  Downloading pytube-15.0.0-py3-none-any.whl.metadata (5.0 kB)
Collecting av==11.* (from faster-whisper==1.0.1->-r requirements.txt (line 4))
  Downloading av-11.0.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (4.5 kB)
Collecting ctranslate2<5,>=4.0 (from faster-whisper==1.0.1->-r requirements.txt (line 4))
  Downloading ctranslate2-4.1.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (10 kB)
Collecting huggingface-hub>=0.13 (from faster-whisper==1.0.1->-r requirements.txt (line 4))
  Downloading huggingface_hub-0.21.4-py3-none-any.whl.metadata (13 kB)
Collecting tokenizers<0.16,>=0.13 (from faster-whisper==1.0.1->-r requirements.txt (line 4))
  Downloading tokenizers-0.15.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (6.7 kB)
Collecting onnxruntime<2,>=1.14 (from faster-whisper==1.0.1->-r requirements.txt (line 4))
  Downloading onnxruntime-1.17.1-cp311-cp311-manylinux_2_27_x86_64.manylinux_2_28_x86_64.whl.metadata (4.3 kB)
Collecting aiofiles<24.0,>=22.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading aiofiles-23.2.1-py3-none-any.whl.metadata (9.7 kB)
Collecting altair<6.0,>=4.2.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading altair-5.2.0-py3-none-any.whl.metadata (8.7 kB)
Collecting fastapi (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading fastapi-0.110.0-py3-none-any.whl.metadata (25 kB)
Collecting ffmpy (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading ffmpy-0.3.2.tar.gz (5.5 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting gradio-client==0.8.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading gradio_client-0.8.0-py3-none-any.whl.metadata (7.1 kB)
Collecting httpx (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading httpx-0.27.0-py3-none-any.whl.metadata (7.2 kB)
Collecting importlib-resources<7.0,>=1.3 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading importlib_resources-6.3.1-py3-none-any.whl.metadata (3.9 kB)
Collecting jinja2<4.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading Jinja2-3.1.3-py3-none-any.whl.metadata (3.3 kB)
Collecting markupsafe~=2.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading MarkupSafe-2.1.5-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (3.0 kB)
Collecting matplotlib~=3.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading matplotlib-3.8.3-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (5.8 kB)
Collecting numpy~=1.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading numpy-1.26.4-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (61 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.0/61.0 kB 1.6 MB/s eta 0:00:00
Collecting orjson~=3.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading orjson-3.9.15-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (49 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 49.5/49.5 kB 1.3 MB/s eta 0:00:00
Collecting packaging (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading packaging-24.0-py3-none-any.whl.metadata (3.2 kB)
Collecting pandas<3.0,>=1.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading pandas-2.2.1-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (19 kB)
Collecting pillow<11.0,>=8.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading https://download.pytorch.org/whl/pillow-10.2.0-cp311-cp311-manylinux_2_28_x86_64.whl (4.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.5/4.5 MB 5.4 MB/s eta 0:00:00
Collecting pydantic>=2.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading pydantic-2.6.4-py3-none-any.whl.metadata (85 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 85.1/85.1 kB 745.9 kB/s eta 0:00:00
Collecting pydub (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading pydub-0.25.1-py2.py3-none-any.whl.metadata (1.4 kB)
Collecting python-multipart (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading python_multipart-0.0.9-py3-none-any.whl.metadata (2.5 kB)
Collecting pyyaml<7.0,>=5.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading PyYAML-6.0.1-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (2.1 kB)
Collecting semantic-version~=2.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading semantic_version-2.10.0-py2.py3-none-any.whl.metadata (9.7 kB)
Collecting tomlkit==0.12.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading tomlkit-0.12.0-py3-none-any.whl.metadata (2.7 kB)
Collecting typer<1.0,>=0.9 (from typer[all]<1.0,>=0.9->gradio==4.14.0->-r requirements.txt (line 6))
  Downloading typer-0.9.0-py3-none-any.whl.metadata (14 kB)
Collecting typing-extensions~=4.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading typing_extensions-4.10.0-py3-none-any.whl.metadata (3.0 kB)
Collecting uvicorn>=0.14.0 (from gradio==4.14.0->-r requirements.txt (line 6))
  Downloading uvicorn-0.28.0-py3-none-any.whl.metadata (6.3 kB)
Collecting fsspec (from gradio-client==0.8.0->gradio==4.14.0->-r requirements.txt (line 6))
  Downloading fsspec-2024.3.0-py3-none-any.whl.metadata (6.8 kB)
Collecting websockets<12.0,>=10.0 (from gradio-client==0.8.0->gradio==4.14.0->-r requirements.txt (line 6))
  Downloading websockets-11.0.3-cp311-cp311-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (6.6 kB)
Collecting filelock (from torch->-r requirements.txt (line 2))
  Downloading filelock-3.13.1-py3-none-any.whl.metadata (2.8 kB)
Collecting sympy (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/sympy-1.12-py3-none-any.whl (5.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5.7/5.7 MB 5.0 MB/s eta 0:00:00
Collecting networkx (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/networkx-3.2.1-py3-none-any.whl (1.6 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.6/1.6 MB 3.7 MB/s eta 0:00:00
Collecting nvidia-cuda-nvrtc-cu11==11.8.89 (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/cu118/nvidia_cuda_nvrtc_cu11-11.8.89-py3-none-manylinux1_x86_64.whl (23.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 23.2/23.2 MB 6.4 MB/s eta 0:00:00
Collecting nvidia-cuda-runtime-cu11==11.8.89 (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/cu118/nvidia_cuda_runtime_cu11-11.8.89-py3-none-manylinux1_x86_64.whl (875 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 875.6/875.6 kB 3.3 MB/s eta 0:00:00
Collecting nvidia-cuda-cupti-cu11==11.8.87 (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/cu118/nvidia_cuda_cupti_cu11-11.8.87-py3-none-manylinux1_x86_64.whl (13.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 13.1/13.1 MB 6.8 MB/s eta 0:00:00
Collecting nvidia-cudnn-cu11==8.7.0.84 (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/cu118/nvidia_cudnn_cu11-8.7.0.84-py3-none-manylinux1_x86_64.whl (728.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 728.5/728.5 MB 2.1 MB/s eta 0:00:00
Collecting nvidia-cublas-cu11==11.11.3.6 (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/cu118/nvidia_cublas_cu11-11.11.3.6-py3-none-manylinux1_x86_64.whl (417.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 417.9/417.9 MB 2.9 MB/s eta 0:00:00
Collecting nvidia-cufft-cu11==10.9.0.58 (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/cu118/nvidia_cufft_cu11-10.9.0.58-py3-none-manylinux1_x86_64.whl (168.4 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 168.4/168.4 MB 3.9 MB/s eta 0:00:00
Collecting nvidia-curand-cu11==10.3.0.86 (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/cu118/nvidia_curand_cu11-10.3.0.86-py3-none-manylinux1_x86_64.whl (58.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 58.1/58.1 MB 5.5 MB/s eta 0:00:00
Collecting nvidia-cusolver-cu11==11.4.1.48 (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/cu118/nvidia_cusolver_cu11-11.4.1.48-py3-none-manylinux1_x86_64.whl (128.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 128.2/128.2 MB 5.0 MB/s eta 0:00:00
Collecting nvidia-cusparse-cu11==11.7.5.86 (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/cu118/nvidia_cusparse_cu11-11.7.5.86-py3-none-manylinux1_x86_64.whl (204.1 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 204.1/204.1 MB 4.0 MB/s eta 0:00:00
Collecting nvidia-nccl-cu11==2.19.3 (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/cu118/nvidia_nccl_cu11-2.19.3-py3-none-manylinux1_x86_64.whl (135.3 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 135.3/135.3 MB 4.3 MB/s eta 0:00:00
Collecting nvidia-nvtx-cu11==11.8.86 (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/cu118/nvidia_nvtx_cu11-11.8.86-py3-none-manylinux1_x86_64.whl (99 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 99.1/99.1 kB 8.6 MB/s eta 0:00:00
Collecting triton==2.2.0 (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/triton-2.2.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (167.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 167.9/167.9 MB 4.2 MB/s eta 0:00:00
INFO: pip is looking at multiple versions of openai-whisper to determine which version is compatible with other requirements. This could take a while.
Collecting torch (from -r requirements.txt (line 2))
  Downloading torch-2.2.1-cp311-cp311-manylinux1_x86_64.whl.metadata (26 kB)
Collecting nvidia-cuda-nvrtc-cu12==12.1.105 (from torch->-r requirements.txt (line 2))
  Downloading nvidia_cuda_nvrtc_cu12-12.1.105-py3-none-manylinux1_x86_64.whl.metadata (1.5 kB)
Collecting nvidia-cuda-runtime-cu12==12.1.105 (from torch->-r requirements.txt (line 2))
  Downloading nvidia_cuda_runtime_cu12-12.1.105-py3-none-manylinux1_x86_64.whl.metadata (1.5 kB)
Collecting nvidia-cuda-cupti-cu12==12.1.105 (from torch->-r requirements.txt (line 2))
  Downloading nvidia_cuda_cupti_cu12-12.1.105-py3-none-manylinux1_x86_64.whl.metadata (1.6 kB)
Collecting nvidia-cudnn-cu12==8.9.2.26 (from torch->-r requirements.txt (line 2))
  Downloading nvidia_cudnn_cu12-8.9.2.26-py3-none-manylinux1_x86_64.whl.metadata (1.6 kB)
Collecting nvidia-cublas-cu12==12.1.3.1 (from torch->-r requirements.txt (line 2))
  Downloading nvidia_cublas_cu12-12.1.3.1-py3-none-manylinux1_x86_64.whl.metadata (1.5 kB)
Collecting nvidia-cufft-cu12==11.0.2.54 (from torch->-r requirements.txt (line 2))
  Downloading nvidia_cufft_cu12-11.0.2.54-py3-none-manylinux1_x86_64.whl.metadata (1.5 kB)
Collecting nvidia-curand-cu12==10.3.2.106 (from torch->-r requirements.txt (line 2))
  Downloading nvidia_curand_cu12-10.3.2.106-py3-none-manylinux1_x86_64.whl.metadata (1.5 kB)
Collecting nvidia-cusolver-cu12==11.4.5.107 (from torch->-r requirements.txt (line 2))
  Downloading nvidia_cusolver_cu12-11.4.5.107-py3-none-manylinux1_x86_64.whl.metadata (1.6 kB)
Collecting nvidia-cusparse-cu12==12.1.0.106 (from torch->-r requirements.txt (line 2))
  Downloading nvidia_cusparse_cu12-12.1.0.106-py3-none-manylinux1_x86_64.whl.metadata (1.6 kB)
Collecting nvidia-nccl-cu12==2.19.3 (from torch->-r requirements.txt (line 2))
  Downloading nvidia_nccl_cu12-2.19.3-py3-none-manylinux1_x86_64.whl.metadata (1.8 kB)
Collecting nvidia-nvtx-cu12==12.1.105 (from torch->-r requirements.txt (line 2))
  Downloading nvidia_nvtx_cu12-12.1.105-py3-none-manylinux1_x86_64.whl.metadata (1.7 kB)
Collecting nvidia-nvjitlink-cu12 (from nvidia-cusolver-cu12==11.4.5.107->torch->-r requirements.txt (line 2))
  Downloading nvidia_nvjitlink_cu12-12.4.99-py3-none-manylinux2014_x86_64.whl.metadata (1.5 kB)
Collecting torch (from -r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/cu118/torch-2.2.0%2Bcu118-cp311-cp311-linux_x86_64.whl (811.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 811.7/811.7 MB 2.0 MB/s eta 0:00:00
  Downloading torch-2.2.0-cp311-cp311-manylinux1_x86_64.whl.metadata (25 kB)
  Downloading https://download.pytorch.org/whl/cu118/torch-2.1.2%2Bcu118-cp311-cp311-linux_x86_64.whl (2325.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.3/2.3 GB 895.4 kB/s eta 0:00:00
Collecting triton==2.1.0 (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/triton-2.1.0-0-cp311-cp311-manylinux2014_x86_64.manylinux_2_17_x86_64.whl (89.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 89.2/89.2 MB 4.7 MB/s eta 0:00:00
Collecting torch (from -r requirements.txt (line 2))
  Downloading torch-2.1.2-cp311-cp311-manylinux1_x86_64.whl.metadata (25 kB)
Collecting nvidia-nccl-cu12==2.18.1 (from torch->-r requirements.txt (line 2))
  Downloading nvidia_nccl_cu12-2.18.1-py3-none-manylinux1_x86_64.whl.metadata (1.8 kB)
Collecting torch (from -r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/cu118/torch-2.1.1%2Bcu118-cp311-cp311-linux_x86_64.whl (2326.0 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.3/2.3 GB 858.2 kB/s eta 0:00:00
  Downloading torch-2.1.1-cp311-cp311-manylinux1_x86_64.whl.metadata (25 kB)
INFO: pip is still looking at multiple versions of openai-whisper to determine which version is compatible with other requirements. This could take a while.
  Downloading https://download.pytorch.org/whl/cu118/torch-2.1.0%2Bcu118-cp311-cp311-linux_x86_64.whl (2325.9 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.3/2.3 GB 891.9 kB/s eta 0:00:00
  Downloading torch-2.1.0-cp311-cp311-manylinux1_x86_64.whl.metadata (25 kB)
  Downloading https://download.pytorch.org/whl/cu118/torch-2.0.1%2Bcu118-cp311-cp311-linux_x86_64.whl (2267.3 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.3/2.3 GB 879.5 kB/s eta 0:00:00
Collecting triton==2.0.0 (from torch->-r requirements.txt (line 2))
  Downloading https://download.pytorch.org/whl/triton-2.0.0-1-cp311-cp311-manylinux2014_x86_64.manylinux_2_17_x86_64.whl (63.3 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 63.3/63.3 MB 4.9 MB/s eta 0:00:00
Collecting cmake (from triton==2.0.0->torch->-r requirements.txt (line 2))
  Downloading cmake-3.28.3-py2.py3-none-manylinux2014_x86_64.manylinux_2_17_x86_64.whl.metadata (6.3 kB)
Collecting lit (from triton==2.0.0->torch->-r requirements.txt (line 2))
  Downloading lit-18.1.1.tar.gz (161 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 161.1/161.1 kB 2.1 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Collecting numba (from openai-whisper==20231106->-r requirements.txt (line 3))
  Downloading numba-0.59.0-cp311-cp311-manylinux2014_x86_64.manylinux_2_17_x86_64.whl.metadata (2.7 kB)
Collecting tqdm (from openai-whisper==20231106->-r requirements.txt (line 3))
  Downloading tqdm-4.66.2-py3-none-any.whl.metadata (57 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 57.6/57.6 kB 771.5 kB/s eta 0:00:00
Collecting more-itertools (from openai-whisper==20231106->-r requirements.txt (line 3))
```

**Comment by jhj0517** [comment]
This should be fixed in #124, 
Please re-open it if the issue persists.

**Comment by aschmitz13** [comment]
Yes this dependency issue appeared with me too. But I have switched to another web front-end which was working smoothly. That's why I haven't followed up. Sorry.
